### PR TITLE
Add take1() to GpuArray.

### PR DESCRIPTION
**Pull request by abergeron**
I just realized that this is missing.

### CODE REVIEW

File: pygpu/gpuarray.pyx [hunk 1]
```diff
@@ -1695,6 +1704,20 @@ cdef class GpuArray:
 
         array_setarray(tmp, gv)
 
+    def take1(self, GpuArray idx):
```
Comment (nouiz) [inline pygpu/gpuarray.pyx]: Add tests for that method. It can be basic test just to make sure it work and we don't remove/break it by mistake.

**Comment by nouiz** [comment]
It is good that you add it. Did you plan to use it in Theano?

**Comment by abergeron** [comment]
Yes probably for the GpuAdvancedSubtensor python code.

**Comment by abergeron** [comment]
Now with test™.

**Pull request merged**